RUMM-1435: Update SDK requirements based on backward compatibility tests (#74)
DataDog/dd-sdk-reactnative: docs/rum_integrations.md

## RUM Integrations

### Integrate with ReactNavigation

> We provide automatic integration for [ReactNavigation](https://reactnavigation.org/) API (minimum supported version is `react-navigation/native@5.6.0`). You just need to add this in your source code:

Reviewer: ```suggestion
Datadog provides automatic integration for the [ReactNavigation](https://reactnavigation.org/) API (minimum supported version is `react-navigation/native@5.6.0`). Add the following in your source code:
```
